src/routes/Routes.tsx

```diff
@@ -149,9 +150,16 @@ const Routes = () => {
         <Route
           path={USER_EDIT_ROUTE}
           element={
-            <UserLayout routeProps={{ hasBackLink: true }}>
+            <Default
+              routeProps={{
+                hasBackLink: true,
+                hasSubscribedColonies: isMobile,
```

Comment:
Is that supposed to be showing colonies list somewhere in the /edit-profile route on mobiles? Couldn't see it anywhere

Reply:
Yes, when the mobile version of the subscribed colonies list gets ported, it'll show. I envision this being resolved by https://github.com/JoinColony/colonyCDapp/issues/99